src/stories/Library/tag/Tag.tsx

```diff
@@ -2,42 +2,44 @@ import { useState } from "react";
 import clsx from "clsx";
 
 type TagProps = {
-  label: string;
-  removable?: boolean;
-  facet?: boolean;
+  children: React.ReactNode;
+  size?: "small" | "large";
+  usesCursor?: boolean;
+  hasBackground?: boolean;
+  showCloseIcon?: boolean;
+  isClickable?: boolean;
 };
 
-const Icon = () => (
-  <img className="tag-icon" src="icons/basic/icon-cross.svg" alt="close icon" />
-);
-
-export const Tag = ({ label, removable = false, facet }: TagProps) => {
+export const Tag = ({
+  children,
+  hasBackground = false,
+  size = "small",
+  usesCursor = true,
+  showCloseIcon = false,
+  isClickable = true,
```

Comment:
Do we have to have both usesCursor + isClickable ?
or could the cursor-pointer class just come on if isClickable

Reply:
By having both props we open up the possibility for having a tag that you can click on but does not change background when clicked.